src/js/xhr.js

```diff
@@ -1,5 +1,6 @@
-import * as VjsUtils from './util';
```

Comment:
Yeah, I was thinking the same thing. We have some specific `file:///` url and `withCredentials` handling that we'll need to know how they translate or at least know we're losing them. But we should definitely make that happen before 5.0. Feel like making a PR for that?